src/manifest.ts

```diff
@@ -204,6 +226,14 @@ export class PluginsUpdater implements ObsidianManifest {
                 // reload plugins after updated
                 (this.app as any).plugins.enablePluginAndSave(plugin.id); // eslint-disable-line @typescript-eslint/no-explicit-any
                 // update notice display
+                this.checkedPlugins++;
+                const spanProgressBar = document.getElementById(`span-plugin-updating-progress-bar`);
+                console.log(spanProgressBar);
+                console.log(this.checkedPlugins);
+                console.log(100 * (this.checkedPlugins / this.totalPlugins));
```

Comment:
debug logging should not use `console.log`, delete them or use this plugin logging interface